yarn.lock

```diff
@@ -1953,6 +1953,33 @@
     tabbable "^1.1.0"
     uuid "^3.1.0"
 
+"@elastic/eui@18.2.1":
```

Comment:
accidental?

Reply:
Yes it was, I fixed it in https://github.com/elastic/kibana/pull/56661.

Reply:
should it be removed from this PR?

Reply:
Hahaha, yes, not sure what happened there. Thanks!